products/facilities/README.md

```diff
@@ -1,221 +1,50 @@
-# db-facilities
+# Facilities database (FacDB)  [![Build](https://github.com/NYCPlanning/data-engineering/actions/workflows/facilities_build.yml/badge.svg)](https://github.com/NYCPlanning/data-engineering/actions/workflows/facilities_build.yml)
```

Comment:
since the FacDB workflow linked here isn't used anymore, we probably shouldn't have this badge

unfortunately there's no way (I can find) to filter our `build.yml` [runs](https://github.com/NYCPlanning/data-engineering/actions/workflows/build.yml) to be specific to a dataset

Reply:
got it! I'm assuming the same is true for ztl, correct?

Reply:
yea [looks like it](https://github.com/NYCPlanning/data-engineering/actions/workflows/zoningtaxlots_build.yml)